content/applications/hr/payroll/payslips.rst

```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
 
-If there are any errors or issues preventing the payslips from being generated, an error message
-appears in the top-right section. This error box disappears on its own after several seconds, or the
-:guilabel:`✖ (x mark)` icon can be clicked to close the alert.
+Click the checkbox to the left of each payslip to select it. When all desired payslips have been
+selected, click the :guilabel:`Select` button. The pop-up window closes, and the payslips have been
+added to the batch.
 
-To remedy the issue, make any necessary modifications (e.g. removing any payslip lines that cannot
-be processed), then click the :guilabel:`Generate` button again.
+.. _payroll/batch-process:
 
-Once the payslips have been successfully generated, the screen returns to the payslip batch form.
+Process a batch
+---------------
 
-From here, click the :guilabel:`Generate Draft Entry` button to change the payslips' status from
-:guilabel:`Draft` to :guilabel:`Done`.
+Click on an individual batch to view the details for that batch on a separate page. On this batch
+detail page, different options (buttons) appear at the top, depending on the status of the batch:
+
+- :guilabel:`New`: batches without any payslips added to them have a status of :guilabel:`New`.
+  Only these batches present the option to add payslips to the selected batch. Click the
+  :guilabel:`Add Payslips` button and an :guilabel:`Add Payslips` pop-up window appears. Only
+  payslips that can be added to the batch (payslips not currently part of a batch) appear on the
+  list. Select the desired payslips by clicking the small box to the left of each payslip name, then
+  click the :guilabel:`Select` button to add them to the batch.
+- :guilabel:`Confirmed`: batches that have been created and have payslips in them, but the payslips
+  have not been paid out yet, have a status of :guilabel:`Confirmed`. These batches display a
+  :guilabel:`Create Draft Entry` button. Click the :guilabel:`Create Draft Entry` button to confirm
+  the individual payslips (and the batch), and create a draft of the payslips. The batch now has a
+  status of :guilabel:`Done`.
+- :guilabel:`Done`: batches with confirmed payslips in them have a status of :guilabel:`Done`. The
+  following two button options appear for these batches:
+
+  - :guilabel:`Create Payment Report`: click the :guilabel:`Create Payment Report` button and a
+    :guilabel:`Select a bank journal` pop-up window appears. Select the correct bank journal from
+    the drop-down menu. The batch name appears in the :guilabel:`File name` field, but this can be
+    modified if desired. Finally, click :guilabel:`Confirm` to process the payslips and pay the
+    employees.
+  - :guilabel:`Mark as paid`: after the payments have been created via the :guilabel:`Create Payment
+    Report` button, the payslips need to be marked as paid in the database. Click the
+    :guilabel:`Mark as paid` button, and the status of the batch changes to :guilabel:`Paid`.
+
```

Comment:
I'm also seeing a 'Create ABA File' button and 'Set to Draft' button -- do you think those need to be explained?

Reply:
I added the set to draft option where it appears- but I never see the ABA one- I think that is for specific accounts with some kind of ABA setting on them and not a 'universal' issue- I am checking with the team and seeing if this needs to be added/explained/is common.

Reply:
OK- these are form specific localization settings and are not universal! =)  For example, SEPA is in Europe only.